packages/influxdb-async/influxdb-async.0.3.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "InfluxDB client library using async for concurrency"
+description: "A client library for writing time series data to InfluxDB"
+maintainer: "Ryan Slade <ryanslade@gmail.com>"
+authors: "Ryan Slade <ryanslade@gmail.com>"
+homepage: "https://github.com/ryanslade/influxdb-ocaml"
+bug-reports: "https://github.com/ryanslade/influxdb-ocaml/issues"
+dev-repo: "git://github.com/ryanslade/influxdb-ocaml.git"
+build: [["dune" "build" "-p" name "-j" jobs]]
+depends: [
+  "dune" {>= "2.0"}
+  "base"
+  "alcotest" {with-test}
+  "influxdb"
```

Comment:
I would also strongly suggest to add `{= version}` to not have to maintain each previous packages everytime the API breaks. It is not mandatory but it is a friendly advice.